cmake/modules/FindOpus.cmake

```
@@ -59,8 +59,7 @@ find_library(Opus_LIBRARY
 mark_as_advanced(Opus_LIBRARY)
 
 find_path(OpusFile_INCLUDE_DIR
-  NAMES opusfile.h
-  PATH_SUFFIXES opus
+  NAMES opus/opusfile.h
```

Not sure about this. It should work for `opus/opusfile.h` anyway, but with this change `opusfile.h without the opus path prefix won't work anymore.

---

There are two files with this name, namely `<opus/opusfile.h>` and `<taglib/opusfile.h>`. Does the removal of the path prefix have side effects?

---

I was unsure if this would cause a conflict, but indeed, it has, and ubuntu appveyor and scons travis builds have failed.
Possibly PATH_SUFFIXES can stay and maybe that fixes the linux build errors. (I don't know how that works really)

---

**Current behaviour:** Check if `opusfile.h` exists in the global include dirs (non-recursively). If it doesn't exist, look for `opus/opusfile.h` instead.
**Changed behaviour:** Check if `opus/opusfile.h` exists in the global include dirs (non-recursively).

The current behaviour should be able to detect a superset of the files that the code posted here is able to detect, so it doesn't add anything new. Can you elaborate what the problem with the current code is?

---

For some reason, the detection of opus/opus.h worked (same file) but the detection of opusfile.h did not when calling cmake from visual studio

---

Ok, maybe try to remove PATH_SUFFIXES and add both to NAMES:

```suggestion
  NAMES opusfile.h opus/opusfile.h
```